src/test/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetterTest.java

```diff
@@ -348,4 +390,78 @@ void getTxInfoToSign_whenTransactionReceiptNotFoundInSubsequentBlock_shouldThrow
             pegoutBtcTransaction
         ));
     }
+
+    @ParameterizedTest
+    @MethodSource("getTxInfoToSignArgProvider")
+    void getTxInfoToSign_whenTransactionReceiptHasPegoutTransactionCreatedEvent_returnOk(
+        int version,
+        byte[] serializedOutpointValues,
+        List<Coin> expectedOutpointValues
+    )
+        throws HSMReleaseCreationInformationException {
```

Comment:
nit: make sure we are formatting all newly introduced code

Reply:
Done